test/compliance/DELETE.test.js

```diff
@@ -3,7 +3,7 @@
 const cds = require('../cds.js')
 
 describe('DELETE', () => {
-  cds.test(__dirname + '/../bookshop')
+  cds.test(__dirname + '/resources')
```

Comment:
there is a similar test in `UPDATE.test.js`

Reply:
I don't see any :)

Reply:
oops, thought https://github.com/cap-js/cds-dbs/pull/329/files was already merged

Reply:
i will change it in the PR once this is in main :)